test/unit/prisma-photon/node_modules/@generated/photon/runtime/prisma

```diff
@@ -1 +0,0 @@
-asdf
```

Comment:
Perhaps you should rename this file from `prisma` to `query-engine-darwin`

Reply:
Yep, as node_modules are gitignored, I had to force add it.